libpages/dns.go

```diff
@@ -48,7 +48,7 @@ func (ErrKeybasePagesRecordTooMany) Error() string {
 	return "more than 1 TXT record are found for " + keybasePagesPrefix
 }
 
-const kbpRecordPrefix = "_keybase_pages."
+const kbpRecordPrefixes = [2]string{"_keybase_pages.", "_keybasepages."}
```

Comment:
Would you mind adding a comment here explaining it’s important not to change the ordering of the two prefixes?